tests/acceptance/features/webUIRestrictSharing/restrictReSharing.feature

```diff
@@ -1,4 +1,6 @@
-@issue-ocis-1317
+@issue-ocis-1317 @skipOnOCIS @issue-6896
+# this scenario is skipped on ocis because it opens share folder which in not possible in OCIS
+# but it works for OC10 see issue https://github.com/owncloud/web/issues/6896 for more detail
```

Comment:
As we have come to conclusion, we restrain ourselves from adding comments in feature files
```suggestion
```

Reply:
I didn't make any changes in this PR after internal discussion